src/ape/api/transactions.py

```diff
@@ -76,6 +77,12 @@ def __str__(self) -> str:
         params = "\n  ".join(f"{k}: {v}" for k, v in data.items())
         return f"{self.__class__.__name__}:\n  {params}"
 
+    @abstractmethod
+    def txn_hash(self) -> HexBytes:
```

Comment:
one note is that in `ReceiptAPI`, the `txn_hash` property is of type `str` (for DB reasons?)
I think it should be fine as `HexBytes` here but just FYI

Reply:
It should be changed to `HexBytes` in `ReceiptAPI` too, @johnson2427 and I were just talking about this inconsistency

Reply:
HexBytes works with me. It is an int in Starknet but it can be translated.